src/java/com/zimbra/qa/selenium/projects/admin/pages/AdminPages.java

```diff
@@ -110,6 +110,13 @@
 	public PageManageMigrationCoexistance zPageManageMigrationCoexistance = null;
 	public PageManageHelp zPageManageHelp = null;
 
+	private static AdminPages AdminPages;
+	public static AdminPages getInstance() {
+		if (AdminPages == null) {
+			AdminPages = new AdminPages();
+		}
+		return AdminPages;
+	}
 	public AdminPages() {
```

Comment:
Make this constructor as a private instead of public.

Reply:
Done.